ai-services/cmd/ai-services/cmd/bootstrap/configure.go

```diff
@@ -136,6 +137,18 @@ func runServiceReport() error {
 		return err
 	}
 
+	cards, err := helpers.FindFreeSpyreCards()
```

Comment:
what if there are no cards returned considering they are consumed by the application, then this I also don't want to run the rmmod,!

can be a simple check where number of lspci entries not matching to vfio entries then rmmod and insmod

Reply:
Done, ptal